mkosi/__init__.py

```diff
@@ -2699,17 +2704,17 @@ def install_arch(args: MkosiArgs, root: Path, do_run_build_script: bool) -> None
                 Architecture = auto
                 Color
                 CheckSpace
-                SigLevel = Required DatabaseOptional TrustAll
+                SigLevel = {"Required DatabaseOptional TrustAll" if not args.use_mirror_verbatim else "Never"}
                 ParallelDownloads = 5
 
                 [core]
                 {server}
 
-                [extra]
-                {server}
+                {"[extra]" if not args.use_mirror_verbatim else ""}
+                {f"{server}" if not args.use_mirror_verbatim else ""}
 
-                [community]
-                {server}
+                {"[community]" if not args.use_mirror_verbatim else ""}
+                {f"{server}" if not args.use_mirror_verbatim else ""}
```

Comment:
I think this would be clearer if split into two different versions of `pacman.conf`, one for the normal case and one for `--use-mirror-verbatim`.

Reply:
rather than two versions, I've split the write in two - there's already a separate write later with optional elements